src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const container = document.querySelector('.container');
```

Comment:
```suggestion
const population = document.querySelectorAll('.population');
const totalPopulation = document.querySelector('.total-population');
const averagePopulation = document.querySelector('.average-population');
```

You can easily get rid of this redundant 'container' and use 'document' instead. You're searching for specific class-names and there no ned to find inside 'container' in this task. Just FYI